src/views/UserDetails.vue

```diff
@@ -5,7 +5,7 @@
         <v-container>
           <suspense>
             <user-card
-              :name="user.name"
+              :name="user.firstName"
```

Comment:
We should put all of the new fields in the UserCard.

Reply:
done